packages/foundation/src/createComponent.tsx

```diff
@@ -114,79 +17,68 @@ export type IStatelessComponent<TComponentProps, TStyleSet extends IStyleSet<TSt
  *
  * State components should contain all stateful behavior and should not generate any JSX, but rather simply call the view prop.
  * Views should simply be stateless pure functions that receive all props needed for rendering their output.
- * State component is optional. If state not provided, created component is essentially a functional stateless component.
+ * State component is optional. If state is not provided, created component is essentially a functional stateless component.
  *
  * TComponentProps: A styleable props interface for the created component.
```

Comment:
Could you use either `@typeparam` or `@template` (or at least a bulleted list) to document these type parameters? This format makes them all show up in intellisense as a single line run together.

Reply:
Which one of these will work? `@typeparam` doesn't seem to exist, and neither does `@template` (or it's hard to search for.) `@param <T>` doesn't seem to work. @kkjeer Do you know if there are any concerns from an API extractor perspective?

Reply:
The TS compiler supports `@template` to declare generic type parameters in JS. 
https://github.com/Microsoft/TypeScript/wiki/JsDoc-support-in-JavaScript#template

There's also an RFC (by Pete I think) to support either template or typeparam in tsdoc, and discussion various other places, but no consensus.
https://github.com/Microsoft/tsdoc/issues/72

Reply:
A bulleted list would also work if we don't want to use a non-finalized tag. :)